automation/deployment/deployer.py

```diff
@@ -411,9 +411,6 @@ def _generate_helm_install_arguments(
             Constants.helm_release_name,
             self._chart_name,
             "--install",
-            "--wait",
```

Comment:
Note that we had to remove the wait argument here, since we are not acquiring external IP address on some K8S SVCs this was waiting for infinite